crypto/x509/x509_lu.c

```diff
@@ -460,6 +467,24 @@ int X509_OBJECT_set1_X509_CRL(X509_OBJECT *a, X509_CRL *obj)
     return 1;
 }
 
+int X509_OBJECT_set0_data(X509_OBJECT *a, char *ptr)
+{
+    if (a == NULL)
```

Comment:
As above, the NULL checks are not needed.  (And if they were, they just call ERR_raise)

Reply:
fixed: NULL checks are removed